edx_solutions_api_integration/courses/urls.py

```diff
@@ -32,7 +32,7 @@
         courses_views.CoursesStaticTabsDetail.as_view()),
     url(r'^{0}/static_tabs/*$'.format(COURSE_ID_PATTERN), courses_views.CoursesStaticTabsList.as_view()),
     url(r'^{0}/completions/*$'.format(COURSE_ID_PATTERN),
-        courses_views.CourseModuleCompletionList.as_view(), name='completion-list'),
+        courses_views.BlockCompletionList.as_view(), name='completion-list'),
```

Comment:
@xitij2000 @jcdyer  Can we instead delete this API completely and change Apros to call the new completion v0 or v1 API directly? I don't want to have APIs in here that offer the same data as APIs in `edx-completion`.

Reply:
I agree with the goal.  Just want to make sure we don't break anything in the process.

Reply:
>Are we sure the mobile apps don't need this?

Good question @jcdyer . I'm pretty sure they don't, as the mobile apps will only be calling APIs that support OAuth2 and public access. It looks like this `CourseModuleCompletionList` API endpoint is a server-to-server API (it is a `SecureListAPIView` which uses API key authentication), so it cannot safely be used from mobile apps. I will also ask the mobile teams on this MCKIN ticket to understand how to answer that question in general.